ynput/OpenPype
# Deadline: Houdini submission settings in OP

## Changelog Description
Make houdini submissions respect pools groups.
This is done by:
1. Make collect pools works with some Houdini families/product types.
2. Make Ayon Houdini submitters get group names from Houdini deadline settings.

## Some Screenshots
![image](https://github.com/ynput/OpenPype/assets/20871534/2f5a2e38-aa98-4535-babd-e7af7fc530c5)
![image](https://github.com/ynput/OpenPype/assets/20871534/c5338fa6-206f-41f9-b4c2-ace4748476ea)
![image](https://github.com/ynput/OpenPype/assets/20871534/78a236fc-46ab-4d25-bb7d-0bd4fc879416)

## Additional Info
This was PR backported from AYON https://github.com/ynput/ayon-core/pull/10

## Testing notes:
1. Launch Houdini
2. Create render/cache instances
3. In publisher, you should see the added attribute defs.
4. Submit to Deadline, you should find the published jobs are properly set.
